docs/content_management/configure_ct_field_settings.md

```diff
@@ -0,0 +1,98 @@
+---
+description: Configure Field settings in Content Types.
+---
+
+# Configure Content Type Fields
+
+When you [modify a Content Type](create_edit_content_types.md) and add or modify Fields, 
+you must configure different settings that control the way [[= product_name =]] 
+treats the contents of each Field. Apart from general settings, certain Field Types, such as for example, [Page](#default-configuration-of-pages) and [Relation](#content-relation-settings) Field Types may have additional settings.
```

Comment:
Apart from general settings, certain Field Types, such as [Page](#default-configuration-of-pages) and [Relation](#content-relation-settings) Field Types may have additional settings.